py/torch_tensorrt/dynamo/partitioning/_global_partitioner.py

```diff
@@ -228,8 +228,22 @@ def partition(
     # Determine partitions based on user specifications and operator support
     # Then, fuse partitions and display overview of supported/unsupported operators
     partitions = partitioner.propose_partitions()
-    fused_graph = partitioner.fuse_partitions(partitions)
-
+    # TODO: confirm with Naren whether this change is required or not
+    # tested both with and without this change, it both works
+    # the only difference is the graph node name, an example is as below:
+    # graph():
+    # %x : [num_users=1] = placeholder[target=x]
+    # %_run_on_acc_0 : [num_users=1] = call_module[target=_run_on_acc_0](args = (%x,), kwargs = {})
```

Comment:
I think we condition on _run_on_acc so the name is important @peri044 to confirm

Reply:
Yes, we condition on that at a couple of places in the code
1) https://github.com/search?q=repo%3Apytorch%2FTensorRT+_run_on_acc+path%3A%2F%5Etests%5C%2Fpy%5C%2Fdynamo%5C%2F%2F&type=code
2) https://github.com/search?q=repo%3Apytorch%2FTensorRT+_run_on_acc+path%3A%2F%5Epy%5C%2Ftorch_tensorrt%5C%2Fdynamo%5C%2F%2F&type=code
Using same prefix for both the partitioners would be better.